topics/metagenomics/tutorials/metagenomics-assembly/tutorial.md

```diff
@@ -0,0 +1,286 @@
+> https://zenodo.org/record/6958893/files/ERR2231570_Read_2.fastqsanger.gz
+> https://zenodo.org/record/6958893/files/ERR2231571_Read_1.fastqsanger.gz
+> https://zenodo.org/record/6958893/files/ERR2231571_Read_2.fastqsanger.gz
+> https://zenodo.org/record/6958893/files/ERR2231572_Read_1.fastqsanger.gz
+> https://zenodo.org/record/6958893/files/ERR2231572_Read_2.fastqsanger.gz
+>  ```
+> 4. Click Start, and then Close
+> 5. When they are ready, rename the datasets
+{: .hands_on}
+
+### Assembly
+Co-assembly of short reads will be the focus of this tutorial. Later in this tutorial, we will also cover individual assembly. 
+To accomplish our objectives, we will use the MetaSPAdes assembly tool. MetaSPADES is an NGS assembler designed specifically for large and complex metagenomics datasets. MetaSPAdes is part of the SPAdes toolkit, which has several assembly pipelines.
+
+There are three challenges to metagenomics assembly: 1) differences in genome coverage between samples, resulting from differences in abundance, 2) the fact that different species often share conserved regions ({%cite kececioglu2001%}), and 3) the presence of multiple strains of a single species ({%cite miller2010%}). Since SPAdes handles non-uniform coverage, it is useful for assembling simple communities, but metaSPAdes also handles other problems, allowing it to assemble complex communities' metagenomes.
+
+#### Co-assembly
+
+> ### {% icon hands_on %} Hands-on: Assembly with MetaSPAdes
+>
+> > ### {% icon comment %} Comment
+> >
+> > In the next steps we use **metaSPAdes** assembler. **MetaSPAdes** is a versatile metagenomic assembler ({%cite nurk2017%}). As input for metaspades it can accept short reads. However, there is an option to use additionally long reads besides short reads to produce hybrid input.
+> {: .comment} 
+>
+> 1. Name a new history with the name “Assembly with MetaSPAdes”
+>
+> 2. Run **MetaSPAdes**{% icon tool %} with parameters:
+>     - {% icon param-file %} *"Pair-end reads input format"*: paired_interlaced
+>     - {% icon param-file %} *"FASTQ file(s): interlaced"*: collection of short reads
+>     - {% icon param-file %} *"Select k-mer detection option"*: manual
+>     - {% icon param-file %} *"K-mer size values"*: 21,33,55,77
+> 3. Rename output to “Output - assembly with MetaSPAdes”
+{: .hands_on}
+
+> ### {% icon details %} Output of MetaSPAdes
+>
+> **MetaSPAdes** outputs contigs and scaffolds.
+Scaffolds are segments of genome sequence reconstructed from end-sequenced whole-genome shotgun clones. Contigs and gaps make up scaffolds. Contigs are contiguous lengths of genomic sequences in which bases are known to a high degree of certainty. The gaps occur when reads from the two sequenced ends of at least one fragment overlap with other reads from two different contigs (as long as the arrangement is otherwise consistent with the contigs being adjacent). It is possible to estimate the number of bases between contigs based on fragment lengths.
```

Comment:
each line in a box needs to start with `>` or it won't be displayed properly

```suggestion
> Scaffolds are segments of genome sequence reconstructed from end-sequenced whole-genome shotgun clones. Contigs and gaps make up scaffolds. Contigs are contiguous lengths of genomic sequences in which bases are known to a high degree of certainty. The gaps occur when reads from the two sequenced ends of at least one fragment overlap with other reads from two different contigs (as long as the arrangement is otherwise consistent with the contigs being adjacent). It is possible to estimate the number of bases between contigs based on fragment lengths.
```